Mage/Task/BuiltIn/Deployment/Strategy/TarGzTask.php

```diff
@@ -132,7 +156,10 @@ public function run()
     protected function excludesListFile($excludesFile)
     {
         $excludesListFileRsync = '';
-        if (!empty($excludesFile) && file_exists($excludesFile) && is_file($excludesFile) && is_readable($excludesFile)) {
+        if (!empty($excludesFile)
+            && file_exists($excludesFile)
+            && is_file($excludesFile)
+            && is_readable($excludesFile)) {
```

Comment:
is_readable() Returns TRUE if the file or directory specified by filename exists and is readable, FALSE otherwise.

I think this overcheck is not needed, furthermore is there any special reason for check that it is a file(why do you avoid links?)?